src/components/list-item/daikin-list-item.ts

```diff
@@ -0,0 +1,126 @@
+import { cva } from "class-variance-authority";
+import { LitElement, css, html, nothing, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import tailwindStyles from "../../tailwind.css?inline";
+import "../icon/daikin-icon";
+import type { IconType } from "../icon/daikin-icon";
+
+const cvaList = cva(
+  [
+    "flex",
+    "justify-between",
+    "items-center",
+    "gap-2",
+    "w-full",
+    "min-h-12",
+    "p-3",
+    "focus-visible:outline",
+    "focus-visible:outline-1",
+    "focus-visible:-outline-offset-1",
+    "focus-visible:outline-daikinBlue-700",
+  ],
+  {
+    variants: {
+      disabled: {
+        false: ["hover:bg-daikinNeutral-100", "active:bg-daikinNeutral-200"],
+        true: ["text-daikinNeutral-200", "[&>*]:text-daikinNeutral-200"],
```

Comment:
Do we need `[&>*]:text-daikinNeutral-200`?

```suggestion
        true: ["text-daikinNeutral-200"],
```